synthesizer/src/vm/execute.rs

```diff
@@ -327,6 +334,79 @@ mod tests {
         }
     }
 
+    #[cfg(feature = "test")]
```

Comment:
I had to use the `feature = "test"` to actually run this test as this: 
```
    #[test]
    fn test_fee_migration_occurs_at_correct_block_height() {
        // Print the CONSENSUS_V2_HEIGHT
        println!("CONSENSUS_V2_HEIGHT: {}", CurrentNetwork::CONSENSUS_V2_HEIGHT);
    }
```
using: `cargo 'test' '--package' 'snarkvm-synthesizer' '--lib' '--' 'vm::execute::tests::test_fee_migration_occurs_at_correct_block_height' '--exact' '--show-output'
`

Always outputs: `CONSENSUS_V2_HEIGHT: 2800000` instead of `CONSENSUS_V2_HEIGHT: 10` which is what I'd expect. I think it's how the `test_helpers` are organized but I'm honestly not positive.